### PR TITLE
Add shape project and refactor for movies project

### PR DESCRIPTION
**Pull request by jquino**
Added shape project
Added refactor code for movies project

### CODE REVIEW

**Comment by carledriss** [comment]
@jquino 
So far looks good!
but it is missing unit tests

File: src/main/java/com/jalasoft/selenium/jose/SeleniumExample.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.stream.Collectors;
+
+/**
+ * Created by Jose Rioja on 3/10/2017.
+ */
+public class SeleniumExample {
```
Comment (carledriss) [inline src/main/java/com/jalasoft/selenium/jose/SeleniumExample.java]: Remove this unnecessary class

File: src/main/java/com/jalasoft/selenium/jose/movies/Customer.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Updated by Jose Rioja on 3/21/2017.
+ */
+class Customer {
+    private String name;
+    private List<Rental> rentals = new ArrayList<>();
```
Comment (carledriss) [inline src/main/java/com/jalasoft/selenium/jose/movies/Customer.java]: you should instantiate ArrayList in the constructor

File: build.gradle [hunk 3]
```diff
@@ -14,6 +14,9 @@ repositories {
 dependencies {
 
     compile group: 'junit', name: 'junit', version: '4.12'
+// https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java
+    compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.3.0'
```
Comment (carledriss) [inline build.gradle]: you don't need to commit this dependency in this project.

**Comment by carledriss** [comment]
LGTM !
only two minor issues.
please update against upstream/develop

File: src/main/java/com/jalasoft/selenium/jose/movies/Customer.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * This method returns a string with the amount owed and frequent renter points earned.
+     * @return returns a string.
+     */
+    public String statement() {
```
Comment (carledriss) [inline src/main/java/com/jalasoft/selenium/jose/movies/Customer.java:41]: I don't see unit tests for this method.

**Comment by carledriss** [comment]
@jquino 
Let me know when you have fixed the observations.

File: src/main/java/com/jalasoft/selenium/jose/movies/Customer.java (hunk 2 above)
Comment (jquino) [reply src/main/java/com/jalasoft/selenium/jose/movies/Customer.java]: I moved the initialization to the constructor.

File: build.gradle (hunk 3 above)
Comment (jquino) [reply build.gradle]: It was already fixed when merging the code.

**Comment by jquino** [comment]
@carledriss 
I have just fixed the observations, please let me know if everything is OK.

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**